md/datetimes-management-tutorial.md

```diff
@@ -0,0 +1,163 @@
+This page gives a step by step procedure for creating a process with forms allowing to choose a date (with or without a time) and display it in another step leveraging each of these data types.  
+
+## Date only
+
+The type **DATE ONLY** is for manipulating a date which need no time information. E.g., a birth date.  
+The storage format for this type is an ISO 8601 formatted Character String (e.g. "2017-01-13").  
+
+### Define your business data
+
+* In the studio, start by opening the business data model editor (Menu "Business Data Model" > "Manage...")
+* Create an object with an attribute of type DATE ONLY
+* Create a new process
+* Then, in the process data tab, create a new business variable of the type of your business object
+
+### Define the contract and task operations or business variable initialization script
+
+* In the process/task execution tab > contract section, define the contract inputs using the Add from data... button
+* Select the business variable you created previously
+* Follow the steps to generate a contract with a COMPLEX input containing a DATE ONLY attribute (keep the option to generate the variable initialization script or the operations (for a task) automatically)
+
+### Generate the form
+
+* Generate the UI designer form by clicking on edit in the form section of the process/task execution tab
+* Since you have a DATE ONLY attribute in the contract inputs, a date picker widget is automatically added to the form  
+There is nothing else to do.
+
+::: info
+The date picker widget output is an ISO 8601 formatted String (e.g. "2017-01-13T00:00:00.000Z"). The time information is always set to midnight UTC and it is dropped at contract level when received by the process instantiation / task execution REST API and converted to a `java.time.LocalTime`.
+:::
+
+### Retrieve the date from the database and display it
+
+In another step form (or in any other page) you may want to retrieve the date and display it.  
+In order to do that:  
+* Create a new task after the existing one
+* Generate its form by clicking on edit in the form section of the task execution tab
+* In the UI Designer, you need first to define a new variable of type external API to retrieve the object containing the date attribute using the context:
+
+![REST API request](images/dates-management-tutorial/tuto-dates-api-request.png) <!--{.img-responsive .img-thumbnail}-->
+
+::: info
+The value returned by the REST API will be an ISO 8601 formatted String.  
+:::
+
+* Drop a date picker widget in the page
+* Set the date picker value with the variable you just created  
+You may prefer to display it in some text. In order to do that:  
+* Drop a text widget in the page
+* Replace the text widget value with an expression using the uiDate filter to format the date the way you want. E.g., `{{employee.birthDate | uiDate:'MM/dd/yyyy'}}` (see the embedded documentation for filter for more information on the different formats)
```

Comment:
1. Drop a **Text** widget into the whiteboard
2. Replace the default **Value** property with an expression using the _uiDate_ filter to format the date the way you want. E.g., `{{employee.birthDate | uiDate:'MM/dd/yyyy'}}` 
For more information on the different formats, see the embedded documentation about filters.